apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.js

```diff
@@ -165,7 +171,7 @@ router.post(
       const assessment_id = res.locals.assessment.id;
       const group_id = req.body.group_id;
       const uids = req.body.add_member_uids;
-      const uidlist = uids.split(/[ ,]+/);
+      const uidlist = uids.split(/[ ,]+/).filter((uid) => !!uid);
```

Comment:
This wasn't strictly a bug, but it did mean that an empty string would show a weird error message like `Failed to add the following users: . Please check if the users exist.`. Now an empty `uids` string is just a no-op.